inc/sitemaps/abstract-class-indexable-sitemap-provider.php

```diff
@@ -107,9 +111,29 @@ public function get_index_links( $max_entries ) {
 			}
 		}
 
+		foreach ( $this->get_non_empty_types() as $object_sub_type ) {
+			if ( ! isset( $page_types[ $object_sub_type ] ) || $page_types[ $object_sub_type ] !== true ) {
```

Comment:
I feel having this as an associative array makes it harder to understand, why not a simple `in_array` using a normal array that all processed object sub types are appended to?